ui/src/app/components/object-detail/object-detail.component.scss

```diff
@@ -57,20 +57,31 @@
 .add-index-column {
   margin-left: 50%;
   padding: 10px;
-  .index-column-list {
+  .column-list {
+    width: 40%;
+  }
+  .add-column-button {
+    width: 40%;
+  }
+}
+.add-pk-column {
```

Comment:
Css needs to be improved, column types are not under 'Type' in spanner. https://drive.google.com/file/d/1zuwEiV-yscZa2l1jHvRDduTEXx6ELlx_/view?usp=sharing&resourcekey=0-3RJsYlnSX1dW4QYUd_7IbA